<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDlg</class>
 <widget class="QDialog" name="SearchDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;Search Analysis:</string>
         </property>
         <property name="buddy">
          <cstring>searchAnalisisLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchAnalisisLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="searchPushButton">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="analysisList"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_types"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okPushButton">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelPushButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>searchAnalisisLineEdit</tabstop>
  <tabstop>searchPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SearchDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SearchDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
